build/tools/chainlink-fulfilling-requests.md

```diff
@@ -0,0 +1,161 @@
+
+## Prerequisites
+
+Once your node is deployed, You will have to fund it with Link token. To do this, we must visit the [Fuji Chainlink Faucet](https://linkfaucet.protofire.io/fuji). Make sure you have FUJI selected as your network, enter your wallet address, and click on the **Send me 10 Test Link** button. You should receive test Link in a few seconds.
+
+We are also going to need testnet AVAX. Use the [AVAX Fuji Testnet Faucet](https://faucet.avax-test.network/) to fund the same address with testnet AVAX.
+
+### Funding your node
+
+Now that our wallet is funded, we are going to send some LINK and AVAX to our node address. You can find your Node address during deployment or in the **Keys** tab of the node operator GUI:
+
+![Node address](../../.github/chainlinkp2-01-Screen_Shot_2021-07-15_at_12.41.44_PM.png)
+
+Copy your node address and send your node LINK **and** AVAX. Using Metamask, it will look like this:
+
+![Node funding](../../.github/chainlinkp2-02-Screen_Shot_2021-07-15_at_12.44.06_PM.png)
+
+The total cost to send LINK will be LINK+gas so make sure you have both enough LINK and AVAX. Refresh the page and view your updated balances. Balances are shown in wei.
+
+![Node balance](../../.github/chainlinkp2-03-Screen_Shot_2021-07-18_at_2.27.42_PM.png)
+
+Although UI will show `ETH balance`, the balance is actually in AVAX. Your node is funded! Now it can start fulfilling requests!
+
+## Oracle deployment
+
+For this tutorial we will harmonize Chainlink contracts with the Avalanche Fuji network. First, We will need to create an Oracle to forward requests from on chain to off chain. Open [Remix](https://remix.ethereum.org/), select **Load from Gist** and paste in code from [here](https://gist.github.com/Julian-dev28/acca3b2a449f02225defc92bc2909f2f).
+
+OR
+
+Create a new file named `OracleAndConsumer.sol`:
+
+```bash
+pragma solidity ^0.6.6;
+
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/Oracle.sol";
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/tests/MockOracle.sol";
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/tests/Consumer.sol";
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/tests/BasicConsumer.sol";
+```
+
+Compile the contract and you should see your repository populate
+
+![Compiled contract](../../.github/chainlinkp2-04-Screen_Shot_2021-07-18_at_12.48.49_PM_2.png)
+
+Select the **ChainlinkClient.sol** file, go to line 25 and input the LINK token address:
+
+```jsx
+address constant private LINK_TOKEN_POINTER = 0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846;
+```
+
+Select and compile the **Oracle.sol** file
+
+Define the LINK token contract address, 0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846
+
+and deploy!
+
```

Comment:
Is line 58 referring to the address we added on Line 50? Seems redundant, it might need clarification.

When I compile Oracle.sol (loaded from your gist), I get error about modifying gist. 

Deploy step should be explained better, not actually clear what needs to be done.

Reply:
I see. This step is redundant but it is needed to ensure the parameters are correct

I loaded the Repo from the Gist and compiled, Worked fine for me. I will try a couple more instances and see what happens

Got it. I'll expand on this point more